tests/queries/0_stateless/01268_procfs_metrics.sh

```diff
@@ -41,4 +41,13 @@ timeout $TIMEOUT bash -c show_processes_func &
 
 wait
 
+# otherwise it can be alive after test
```

Comment:
It is better to:
- run `read_numbers_func` under `clickhouse_client_timeout`
- run `show_processes_func` under `clickhouse_client_loop_timeout`
That way client will wait the query.

Also you can use `clickhouse_test_wait_queries` at the end of the test to wait unfinished queries, but this should not be required.

Reply:
I'll merge this one and fix it in another PR.